mipengine/node/node_logger.py

```diff
@@ -1,17 +1,60 @@
+import inspect
+import logging
 import time
 from functools import wraps
 
-from celery.utils.log import get_task_logger
+from mipengine.node import config as node_config
 
 
-def getLogger():
-    return get_task_logger("node")
+def init_logger(ctx_id=None):
```

Comment:
is ctx_id the context_id? if yes please name it accordingly. Naming it ctx_id can only create confusion.